ste/mgr-JobMgr.go

```diff
@@ -152,6 +152,7 @@ func (jm *jobMgr) logConcurrencyParameters() {
 type jobMgrInitState struct {
 	securityInfoPersistenceManager *securityInfoPersistenceManager
 	folderCreationTracker          common.FolderCreationTracker
+	folderDeletionManager          common.FolderDeletionManager
 }
```

Comment:
Glad to see the use this is getting. It's a wonderful little way for us to globally track state without querying every item to do so.